apps/browser/qml/pages/components/HistoryList.qml

```diff
@@ -16,6 +17,7 @@ SilicaListView {
     id: view
     property string search
     property bool showDeleteButton
+    property bool menuClosed
```

Comment:
I don't think why it needed here, menuClosed is !menuOpen, it can be used "menuOpen" property